scripts/styleguide.config.js

```diff
@@ -115,6 +115,10 @@ const allSections = [
                 description: 'Implements the "media object" layout',
                 usageMode: 'expand',
             },
+            {
+                name: 'Headings',
```

Comment:
I think we should replace the existing Typography section instead of creating a new one for these header components, since they are essentially redundant. That would also be a good place for showing more complex combinations of the headers and body text.

Reply:
I was thinking of replacing but we still have the style though. So I created the new sections. I can update that instead and make the old one says deprecated or something